o2a/o2a_libs/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "o2a_lib"
+version = "0.0.1"
```

Comment:
Please make the version the same as the o2a tool.

Reply:
@michalmodras I have changed the version to 2.0.0, also I changed the version for the main package to the 2.0.0